scalableminds/webknossos
# Remove filters for user list

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create TeamB (defaultTeam, sampleAdmin and sampleNoAdmin already exist)
- Create userA (team manager in teamB, team member in defaultTeam)
- Create userB (team member in defaultTeam, team member in teamB)
-----
- Execute the query (/users with sampleNoAdmin token) -> See only yourself
- Set isEditable = true -> []
- Set isEditable=false -> sampleNoAdmin
- Set sampleNoAdmin as Team manager of defaultTeam
- Execute query -> All 4 users
- Set isEditable = true -> All 4 users
- Set isEditable = false -> []
- Set isTeamManagerOrAdmin = true -> Everyone but userB
- Set isTeamManagerOrAdmin = false -> only userB
- Set isAdmin = true -> only sampleAdmin
- Set isAdmin = false -> everyone but sampleAdmin

I don't really understand what the first filter is supposed to do, since we only get to see users we can edit anyway? It can only be used to filter out the requesting user itself when they are not team manager. Doesn't seem useful.

Also, this is behavior that could be well tested, but I am not sure how to write a test for that.

### Issues:
- fixes #5891

------
(Please delete unneeded items, merge only when none are left open)
